packages/lib/OgImages.tsx

```diff
@@ -127,7 +127,6 @@ export const Meeting = ({ title, users = [], profile }: MeetingImageProps) => {
   const avatars = attendees
     .map((user) => {
       if ("image" in user && user?.image) return user.image;
-      if ("username" in user && user?.username) return `${CAL_URL}/${user.username}/avatar.png`;
```

Comment:
**The reason I think this line might be the issue is 👇**

Let's use one of our org event as an example. https://i.cal.com/support/intercom-support?date=2024-11-25&month=2024-11

This shows this broken preview image:
<img width="586" alt="Screenshot 2024-11-23 at 5 05 36 AM" src="https://github.com/user-attachments/assets/648ce97e-522c-4f46-917d-3112ddbaaa98">

If we inspect the page and see the `<meta/>` element for `og:image`, we have this URL:
```
https://cal.com/_next/image?w=1200&q=100&url=%2Fapi%2Fsocial%2Fog%2Fimage%3Ftype%3Dmeeting%26title%3DIntercom%2520Support%26meetingProfileName%3DSupport%26meetingImage%3Dhttps%253A%252F%252Fcal.com%252Fapi%252Favatar%252Fb0b58752-68ad-4c0d-8024-4fa382a77752.png%26names%3DMilos%2520Puac%26usernames%3Dmilos
```

which throws 502 cloudflare error. But if we remove `%26usernames%3Dmilos`, then it renders well:

```
https://cal.com/_next/image?w=1200&q=100&url=%2Fapi%2Fsocial%2Fog%2Fimage%3Ftype%3Dmeeting%26title%3DIntercom%2520Support%26meetingProfileName%3DSupport%26meetingImage%3Dhttps%253A%252F%252Fcal.com%252Fapi%252Favatar%252Fb0b58752-68ad-4c0d-8024-4fa382a77752.png%26names%3DMilos%2520Puac
```